.circleci/config.yml

```diff
@@ -378,7 +420,10 @@ jobs:
                 fabric_build_secret:$FABRIC_BUILD_SECRET \
                 fabric_groups:$FABRIC_GROUPS \
                 ci_build_num:$CIRCLE_BUILD_NUM \
-                commit_sha:$CIRCLE_SHA1
+                commit_sha:$CIRCLE_SHA1 \
```

Comment:
We also have to upload 'sourcemaps' for iOS, this is handled inside the Fastlane file